SharedLibs/TileMapHelper/src/tiled/TiledTileMapLoader.ts

```diff
@@ -0,0 +1,162 @@
+import { integer, PolygonVertices } from "../model/CommonTypes";
+import {
+  EditableTileMap,
+  TileDefinition,
+  TileObject,
+} from "../model/TileMapModel";
+import { TiledMap } from "./Tiled";
+import {
+  extractTileUidFlippedStates,
+  decodeBase64LayerData,
+} from "./TiledLoaderHelper";
+
+/**
+ * It creates a {@link EditableTileMap} from a Tiled JSON.
+ */
+export class TiledTileMapLoader {
+  static load(pako: any, tiledMap: TiledMap): EditableTileMap | null {
```

Comment:
Any reason why parseAtlas, which also is related to TiledMap, is not in this file?

Reply:
`parseAtla` is is dependent to PIXI and use very few data from the Tiled file. I wanted to keep things separated and wait for a new format to see if we can extract an interface to allow atlas cutting. Because, I guess that having both a parser and a render dependency in the same code is a bad idea.